server-phoenix/test/helios_web/clients/tweet_test.exs

```diff
@@ -235,7 +235,7 @@ defmodule HeliosWeb.Clients.TweetTest do
   test "from_api retweet status" do
     assert Tweet.from_api(test_tweet_retweet()) ===
              %HeliosWeb.Clients.Tweet{
-               created_at: "Fri Mar 18 15:56:28 +0000 2022",
+               created_at: ~U[2022-03-18 15:56:28Z],
```

Comment:
Its fine to squash the unit test change with the type change since they're tightly related e.g. tests won't pass in the commit with the type change.